### PR TITLE
Misaliged button fixes #1120

### LINKED ISSUE #1120: Show page buttons misaligned
beneath the photo, there are social share icons and three buttons. the icons look like they have spread out somehow.  likewise, the buttons are now also taking up the full width of the div and pressing against the content sections awkwardly. Screesnshot below:

screen shot 2017-11-28 at 6 08 04 pm

### PR DESCRIPTION
**Pull request by StlMaris123**
In your PR did you:
This pr fixes the misaligned button group on cases show page
  - [x] Include a description of the changes?
  - [x] Mention the issue the PR addresses?
  - [x] Include screenshots of any changes to the UI?
  - [ ] Isolate any changes to gems (meaning that any new, updated, or removed gems and resulting code changes should be in their own PR)?
  - [x] Add and/or update specs for your code?
screenshot from 2018-05-24 18-02-38

### CODE REVIEW

**Comment by rlgreen91** [comment]
Can you add a screenshot of how it looks when it's logged in?  This looks very nice.

File: app/controllers/cases_controller.rb [hunk 1]
```diff
@@ -88,9 +88,8 @@ def destroy
   end
 
   def history
-    unless @this_case.blank? || @this_case.versions.blank?
-      @case_history = @this_case.try(:versions).order(created_at: :asc)
-    end
+    @case_history = @this_case.try(:versions).sort_by(&:created_at).reverse unless
```
Comment (rlgreen91) [inline app/controllers/cases_controller.rb]: Why did you change this back to using `.sort_by`?  I just merged the change to do the reverse at the database level.

Comment (StlMaris123) [reply app/controllers/cases_controller.rb]: fixed, it
